.github/workflows/tests_docker.yml

```
@@ -59,7 +60,11 @@ jobs:
       if: matrix.user == 'root'
       run: docker exec playwright-docker-${{ matrix.tag }}-test npx playwright install chrome msedge
     - name: Run "npm run test" inside docker
+      if: matrix.headless == 'headless'
       run: docker exec -e INSIDE_DOCKER=1 -e CI=1 playwright-docker-${{ matrix.tag }}-test xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" npm run test
+    - name: Run "HEADFUL=1 npm run test" inside docker
+      if: matrix.headless == 'headful'
```

```suggestion
      if: !matrix.headless
```
this could work, or maybe we need:

```suggestion
      if: matrix.headless == false
```